force-app/main/default/pages/MA_SetupAuthWizardPage.page

```diff
@@ -233,7 +225,7 @@ License: BSD 3-Clause License
                         .then( function( result ) {
                             validateMetadataSaveResult( 'ConnectedApp', result );
                             showMessage( 'Connected App created with name "' + fullName + '"', 'SUCCESS' );
-                            return conn.metadata.create( 'AuthProvider', buildAuthProvider( label, fullName, orgDomainURL, consumerKey, encryptedConsumerSecret ) );
+                            return conn.metadata.create( 'AuthProvider', buildAuthProvider( label, fullName, orgDomainURL, consumerKey, consumerSecret ) );
```

Comment:
🔒 If we no longer have to pass in the encrypted consumer secret but rather we can pass in the plain text, then we should generate a strong, random secret on line 206 like we do with the consumer key on line 202